src/route/dataset.ts

```diff
@@ -299,12 +319,66 @@ router.get('/:dataset_id/revision/by-id/:revision_id', async (req: Request, res:
     res.json(dto);
 });
 
+// POST /api/dataset/:dataset_id/revision/id/:revision_id/import
+// Creates a new import on a revision.  This typically only occurs when a user
+// decides the file they uploaded wasn't correct.
+router.post(
+    '/:dataset_id/revision/by-id/:revision_id/import',
```

Comment:
Do we want/need the `by-id` part of the path?

Reply:
For revision yes... For sure... You might want to look up a revision by index... Equally you might want to look up a dataset by code or name.  Same applies to dimension as well... for import it just becomes about consistency.

Reply:
👍